skimage/future/manual_segmentation.py

```diff
@@ -212,6 +227,11 @@ def _on_lasso_selection(vertices):
     labels = (_mask_from_vertices(vertices, image.shape[:2], i)
               for i, vertices in enumerate(list_of_vertex_lists, start=1))
     if return_all:
```

Comment:
Update `return_all` parameter description for both functions to include also info on contours.